content/ru/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -0,0 +1,233 @@
+Если разница превышает этот предел, он пропустит  операцию.
+
+Например, если установлено значение `200`, это позволяет создать задание на 200 секунд после фактического расписания.
+
+### Правила параллелизма
+
+Поле `.spec.concurrencyPolicy` также является необязательным.
+Оно определяет, как обращаться с одновременным выполнением заданий, созданных этим CronJob.
+В этой спецификации может быть указана только одна из следующих директив параллелизма:
+
+* `Allow` (по умолчанию): CronJob разрешает одновременное выполнение заданий.
+* `Forbid`: CronJob не разрешает одновременное выполнение заданий; если пришло время для выполнения нового задания,
+  а предыдущее еще не завершилось, CronJob пропускает новое задание.
+* `Replace`: Если пришло время для выполнения нового задания, а предыдущее еще не завершилось,
+  CronJob заменит текущее задание на новое.
+
+Заметьте, что принцип параллелизма применяется только к заданиям, созданным одним и тем же Cronjob'ом.
+Если есть несколько CronJob'ов, их соответствующие задания всегда могут выполняться одновременно.
+
+### Остановка графика (Schedule suspension)
+
+Можно приостановить выполнение CronJob заданий, присвоив необязательному полю `.spec.suspend` значение true.
+По умолчанию это поле имеет значение false.
+
+Эта настройка _не_ влияет на задания, которые уже были запущены CronJob'ом.
+
+Если установить это поле как true, все последующие выполнения будут приостановлены
+(они остаются запланированными, но CronJob контроллер не запускает задания для выполнения),
+пока вы не отмените приостановку CronJob'a.
+
+{{< caution >}}
+Работы, приостановленные во время запланированного выполнения, считаются пропущенными.
+Когда `.spec.suspend` изменяется с `true` на `false` для существующего CronJob'a без
+[дэдлайна задержки начала задания](#starting-deadline), пропущенные задания планируются немедленно.
+{{< /caution >}}
+
+### Лимиты на историю заданий
+
+Поля `.spec.successfulJobsHistoryLimit` и `.spec.failedJobsHistoryLimit` являются опциональными.
+В этих полях указывается, сколько завершенных и неудачных заданий должно быть сохранено.
+По умолчанию они установлены на 3 и 1 соответственно.  Установка предела в `0` соответствует сохранению
+ни одного из заданий соответствующего типа после их завершения.
+
+Другой способ автоматической очистки заданий см. в разделе [Автоматическая очистка завершенных заданий](/docs/concepts/workloads/controllers/job/#clean-up-finished-jobs-automatically).
+
+### Часовые пояса
+
+{{< feature-state for_k8s_version="v1.27" state="stable" >}}
+
+Для CronJob'a, в которых не указан часовой пояс, {{< glossary_tooltip term_id="kube-controller-manager" text="kube-controller-manager" >}}
+интерпретирует расписания относительно своего локального часового пояса.
+
+Можно указать часовой пояс для CronJob'a, присвоив `.spec.timeZone` имя действительного
+[часового пояса](https://ru.wikipedia.org/wiki/Список_часовых_поясов_по_странам)
```

Comment:
TZ identifiers for the time zones are missing in Russian Wikipedia. Therefore, we'd better use the original link here.
```suggestion
[часового пояса](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).
```